packages/@sanity/form-builder/src/inputs/PortableText/text/TextBlock.styles.ts

```diff
@@ -99,8 +99,9 @@ function textBlockStyle(props: TextBlockStyleProps & {theme: Theme}) {
 
 export const TextRoot = styled.div<TextBlockStyleProps>(textBlockStyle)
 
-export const TextBlockFlexWrapper = styled(Flex)`
+export const TextBlockFlexWrapper = styled(Box)`
   position: relative;
+  display: flex;
```

Comment:
This CSS will not be applied to the `Box` – so right now `TextBlockFlexWrapper` is a `Box` with `position: relative`. And maybe that's the fix? 🙂

But if you want `display: flex` on a `Box` you need to either:

- Use the `display` prop: `<Box display="flex"/>`
- Override display like this:

```
const FlexBox = styled(Box)`
  &:not([hidden]) {
    display: flex;
  }

```